docs/modules/tutorials/pages/tech-basics-for-newbies.adoc

```diff
@@ -0,0 +1,101 @@
+= Trainings
+:noindex:
+== VIVIDUS usage related technical basics
+
+Depending on which plugins/steps you are planning to use, it may be useful to complete the following technical basics trainings before processing to VIVIDUS:
+
+[cols="1,2,2", options="header"]
+|===
+
+|Topic
+|Goal
+|Resources
+
+|Git
+
+a|Get basic understand of managing the version control process with Git:
+
+    * repo cloning
+    * branching strategy, create/update/delete branches
+    * create/update/delete/push/merge/squash a commit, etc.
+    * merge conflicts resolution
+
+a| * https://git-scm.com/about
+
+* https://guides.github.com/introduction/git-handbook/
+
+|Locators
+a|Get basic knowledge & practical experience in:
+
+    * web locators (ID, CSS selectors, xPath expressions)
+
+    * DSL (JsonPath, XML xPath)
+
+
+a| * https://www.guru99.com/locators-in-selenium-ide.html
+
+* https://www.w3schools.com/xml/xml_xpath.asp
+
+[NOTE]
+====
+You can use the following online services for your expressions validation:
+
+    * https://www.freeformatter.com/xpath-tester.html[XPath validator]
+    * https://jsonpath.com/[JsonPath evaluator] (however, 100% compatibility not guaranteed, better tool TBD)
+====
+
+|Query Languages & Data Formats
+
+a|Get basic knowledge & practical experience in:
+
+  * SQL
+  * regular expressions syntax
+
+a| * https://www.w3schools.com/sql
+
+* https://www.regular-expressions.info/
+
+[NOTE]
+====
+Please use the following online service for your expressions validation (other engines may process the same expression in a slightly different way) :
+
+* https://regex101.com/[Regex Validator]
+====
+
+|HTTP and Web Services
+a|Get understanding of:
+
+    * basic REST HTTP requests: POST, GET, PUT, and DELETE
+    * request/response validation
+a|* https://www.tutorialspoint.com/http/index.htm
+
+* https://www.guru99.com/api-testing.html
+
+|CI/CD
+a|Get understanding of:
+
+    * CI/CD process
+    * Popular CI tools
+a| * https://www.redhat.com/en/topics/devops/what-is-ci-cd
+
+* https://smartbear.com/blog/top-continuous-integration-tools-for-devops/
+
+|Test Automation Principles
```

Comment:
I think `Test Automation Principles` and `Test Automation Approaches` should go very first, even before `Git`